### DISCUSSION TITLE
Pretty URLs without trailingslash

### DISCUSSION THREAD

**Discussion by carstenfeller** [GetPublii/Publii#1737] [Help] [answered]
Hi there,
I prefer post / article URLs without a trailing slash, like domain. com/blog/blogpost or domain. com/blog/category/blogpost2
A "listing" or "folder" URL like domain. com/blog/ or domain. com/tags/ should end with a trailing slash.
Enabling the "Pretty URL" feature always adds a /.
Could this be fine tuned somewhere in Publii options?
Thanks.
C.F.

**Comment by bobmitro** [comment, marked as answer]
It's not possible; you can use rewrite rules, such as in .htaccess, but Publii will generate a slash at the end, so the sitemap, etc., will always include it.

From a user perspective, adding a slash at the end of a web address usually doesn't matter. Both versions look the same; most people won't notice the difference.

However, there's a small technical detail to consider. Sometimes, a web server might take a little longer to load a page if you don't include the slash. This is because the server might have to check if the address refers to a file or a folder. If the server isn't set up correctly, the page load could be a bit slower.

**Reply by carstenfeller** [reply]
Thanks Bob,
it was no user or SEO thing, just my workflows and content management before publishing that need some adjustments then...
